addons/website_slides/models/slide_channel.py

```diff
@@ -173,8 +173,8 @@ def _compute_is_member(self):
         for cp in channel_partners:
             result.setdefault(cp.channel_id.id, []).append(cp.partner_id.id)
         for channel in self:
-            channel.valid_channel_partner_ids = result.get(channel.id, False)
-            channel.is_member = self.env.user.partner_id.id in channel.valid_channel_partner_ids if channel.valid_channel_partner_ids else False
+            valid_channel_partner_ids = result.get(channel.id, False)
+            channel.is_member = self.env.user.partner_id.id in valid_channel_partner_ids if valid_channel_partner_ids else False
```

Comment:
Could even be `channel.is_member = self.env.user.partner_id.id in result.get(channel.id, [])` to make it a lot shorter no?